paddle/fluid/framework/fleet/gloo_wrapper.cc

```diff
@@ -225,6 +240,7 @@ void ParallelConnectContext::connectFullMesh(
 
   std::vector<std::shared_ptr<std::thread>> connect_threads(thread_num_);
   // Connect every pair
+  VLOG(0) << "connect_thread_num: " << thread_num_ << ", size: " << size;
```

Comment:
just print this line in rank 0?

Reply:
if print this only in rank 0, it will be not shown in other node's logs.  and it will be only printed twice in the whole init process.